File: src/main/java/com/example/sk/administration/database/domain/TableInfo.java
package com.example.sk.administration.database.domain;

import org.jspecify.annotations.NullMarked;
import org.jspecify.annotations.Nullable;

@NullMarked
public record TableInfo(String tableName,
                        String columnName,
                        String dataType,
                        @Nullable String targetTable,
                        @Nullable String targetColumn) {
}

File: src/main/kotlin/com/example/sk/administration/database/domain/TableInfo.kt
package com.example.sk.administration.database.domain

import org.jspecify.annotations.NullMarked

@NullMarked
data class TableInfo(
    val tableName: String,
    val columnName: String,
    val dataType: String,
    val targetTable: String?,
    val targetColumn: String?
)